src/main/java/lotto/view/constant/ViewConstants.java

```diff
@@ -0,0 +1,15 @@
+package lotto.view.constant;
+
+public class ViewConstants {
+    public static final String INPUT_PURCHASE_AMOUNT = "구입 금액을 입력해 주세요.";
+    public static final String INPUT_WINNING_NUMBERS = "\n당첨 번호를 입력해 주세요 (쉼표로 구분).";
+    public static final String INPUT_BONUS_NUMBER = "\n보너스 번호를 입력해 주세요.";
+    public static final String PURCHASED_LOTTO_COUNT = "\n%d개를 구매했습니다.";
+    public static final String LOTTO_NUMBERS = "%s";
+    public static final String WINNING_STATISTICS = "\n당첨 통계";
+    public static final String DIVIDER = "---";
+    public static final String EARNINGS_RATE = "총 수익률은 %s%%입니다.\n";
+    public static final String WINNING_STATISTICS_FORMAT = "%s (%s원) - %d개\n";
+    public static final String OUTPUT_FORMATTER = "#,###";
+    public static final String EARNINGS_FORMATTER = "#.##";
+}
```

Comment:
```suggestion
package lotto.view.constant;

public class ViewConstants {
    public static final String INPUT_PURCHASE_AMOUNT = "구입 금액을 입력해 주세요.";
    public static final String INPUT_WINNING_NUMBERS = "\n당첨 번호를 입력해 주세요 (쉼표로 구분).";
    public static final String INPUT_BONUS_NUMBER = "\n보너스 번호를 입력해 주세요.";
    public static final String PURCHASED_LOTTO_COUNT = "\n%d개를 구매했습니다.";
    public static final String LOTTO_NUMBERS = "%s";
    public static final String WINNING_STATISTICS = "\n당첨 통계";
    public static final String DIVIDER = "---";
    public static final String EARNINGS_RATE = "총 수익률은 %s%%입니다.\n";
    public static final String WINNING_STATISTICS_FORMAT = "%s (%s원) - %d개\n";
    public static final String OUTPUT_FORMATTER = "#,###";
    public static final String EARNINGS_FORMATTER = "#.##";
    
    private ViewConstants(){}
}
```
이와 같이 클래스 외부에서 객체 생성을 막기위해 생성자를 `private`으로 설정해주는것도 좋을 것 같아요!!